appveyor.yml

```diff
@@ -0,0 +1,12 @@
+version: '{branch}-1.0.{build}'
```

Comment:
Hmm maybe not so simple:

https://github.com/appveyor/ci/issues/691

-[Unknown]